src/SmartComponents/Inventory/applications/general-information.scss

```diff
@@ -0,0 +1,18 @@
+.ins-active-app-general_information {
+    margin-top: 2em;
+
+    .pf-c-content {
+        > div:first-child {
+            border-bottom: solid 0.1em var(--pf-global--primary-color--100)
+        }
+
+        div:not(:first-child) {
+            span:first-child {
+                font-weight: var(--pf-global--FontWeight--bold);
+            }
+            span:not(:first-child) {
+                float: right;
```

Comment:
No floats!!!
Let's use a flex property to accomplish the same thing.

Reply:
If I'm gonna use Level there's not gonna be any need for floats.